### ISSUE TITLE
[Question]. Net 6 will merge with Xamarin? 

### ISSUE THREAD

**Issue by transis2** [xamarin/Xamarin.Forms#12980]
I have a Question about the future o existing xamarin codebase.

When. Net 6 arrive, the code of this repository will be merged with MAUI or they will begin from scratch ?

What happens with all bug fixes, unit testes and improve stuffs? They will be ported or the project already was "forked" and MAUI is growing independent?

**Comment by GalaxiaGuy** [comment]
I do not work for Microsoft or Xamarin, but hopefully I will be able to clarify some points.

Xamarin.Forms != Xamarin.

Xamarin is a set up tools and libraries to allow building Android, iOS and macOS application using the native APIs with C# and .NET (as well as F# and others).
Xamarin Forms is a cross platform framework for building applications. It supports Android, iOS and macOS using Xamarin. It also supports Windows using .NET, Linux using Mono and Tizen using... something.

For .NET 6 it looks like Xamarin as a brand will be going away. Building Xamarin apps will become more like building other .NET apps in a few ways (sharing more of the project system) but in many ways will be the same as today.

Xamarin Forms will be superseded by a project called the Multi-platform App UI (MAUI: https://github.com/dotnet/maui). This will be an evolution of Xamarin Forms and although some breaking changes are likely (and in some case desirable), it is intended that it will be a straightforward transition. Some of the work for that is already happening in this repo.

I doubt any repos will be merged, since ".NET" is already spread across several repos.

**Comment by transis2** [comment]
Thank you @GalaxiaGuy 

My question is about bug fixes and efforts with this project.

**Comment by davidortinau** [comment]
> When. Net 6 arrive, the code of this repository will be merged with MAUI or they will begin from scratch ?
> 
> What happens with all bug fixes, unit testes and improve stuffs? They will be ported or the project already was "forked" and MAUI is growing independent?

The .NET MAUI work including .NET 6 support is happening in the main-handlers branch on this repository and is sync'd weekly to the dotnet/maui repository. This is not a "from scratch" rewrite. The renderers are being ported to the new mapper/handler architecture, decoupling them from the abstraction layer, and "slimming" them by virtue of removing wrapping views.

At some point the work may shift to the dotnet/maui repository and updates will flow the other direction to this repository. It'll all depend on what is the best workflow to maximize efficiency for the team. 

In summary, for a period of time both Xamarin.Forms and .NET MAUI will receive updates and share bug fixes.
